challenge_presentation/src/challenge_presentation/presentation.py

```diff
@@ -86,23 +90,30 @@ def execute(self, userdata=None):
-        function_list.append(partial(self.robot.rightArm._send_joint_trajectory, [[0, 0, 0, 1.7, 0, 0, 0]]))
-        function_list.append(partial(self.robot.leftArm.send_gripper_goal, "open"))
-        function_list.append(partial(self.robot.leftArm.send_gripper_goal, "close"))
-        function_list.append(partial(self.robot.rightArm.send_gripper_goal, "open"))
-        function_list.append(partial(self.robot.rightArm.send_gripper_goal, "close"))
-        function_list.append(partial(self.robot.leftArm.reset))
-        function_list.append(partial(self.robot.rightArm.reset))
-        function_list.append(partial(self.robot.rightArm.wait_for_motion_done))
+        if self.robot.robot_name == "amigo":
+            function_list.append(partial(self.robot.speech.speak, self.trans.TWO_ARMS, language=self.language, voice=self.voice, block=False))
+            function_list.append(partial(self.robot.speech.speak, self.trans.HUMAN_ARMS, language=self.language, voice=self.voice, block=False))
+            function_list.append(partial(self.robot.leftArm.send_joint_trajectory, "wave_front"))
+            function_list.append(partial(self.robot.speech.speak, self.trans.END_OF_ARMS, language=self.language, voice=self.voice, block=False))
+            function_list.append(partial(self.robot.speech.speak, self.trans.GRIPPERS, language=self.language, voice=self.voice, block=False))
+            function_list.append(partial(self.robot.leftArm._send_joint_trajectory, [[0, 0, 0, 1.7, 0, 0, 0]]))
+            function_list.append(partial(self.robot.rightArm._send_joint_trajectory, [[0, 0, 0, 1.7, 0, 0, 0]]))
+            function_list.append(partial(self.robot.leftArm.send_gripper_goal, "open"))
+            function_list.append(partial(self.robot.leftArm.send_gripper_goal, "close"))
+            function_list.append(partial(self.robot.rightArm.send_gripper_goal, "open"))
+            function_list.append(partial(self.robot.rightArm.send_gripper_goal, "close"))
+            function_list.append(partial(self.robot.leftArm.reset))
+            function_list.append(partial(self.robot.rightArm.reset))
+            function_list.append(partial(self.robot.rightArm.wait_for_motion_done))
+        else:
+            function_list.append(partial(self.robot.speech.speak, self.trans.NO_ARMS, language=self.language, voice=self.voice, block=False))
+
 
         # Torso
-        function_list.append(partial(self.robot.speech.speak, self.trans.TORSO, language=self.language, voice=self.voice, block=False))
+        if self.robot.robot_name == "amigo":
+            function_list.append(partial(self.robot.speech.speak, self.trans.TORSO, language=self.language, voice=self.voice, block=False))
+        else:
+            function_list.append(partial(self.robot.speech.speak, self.trans.TORSO_SERGIO, language=self.language, voice=self.voice, block=False))
         function_list.append(partial(self.robot.torso.medium))
```

Comment:
Dit is ook voor SERGIO nog helemaal niet geimplementeerd, toch (d.w.z., de torso gaat helemaal niet bewegen...)? Wellicht ook even uitcommenten...